### DISCUSSION TITLE
Unable to access student developer pack

### DISCUSSION THREAD

**Discussion by 0x7771** [community/community#52224] [GitHub Education]
Hello guys,

I got the student program verified and approved but I cannot access to student developer pack.

<img width="344" alt="image" src="https://user-images.githubusercontent.com/6846230/230538376-8a9b9a32-8bcc-482a-ac95-253f3d61885a.png">

<img width="936" alt="image" src="https://user-images.githubusercontent.com/6846230/230538330-6178a014-67e8-4c1b-a556-cc15270fa78b.png">

After click on Student Developer Pack, it goes to "are you student" page.

<img width="762" alt="image" src="https://user-images.githubusercontent.com/6846230/230538305-15da133c-3e29-4339-90de-9a461eff2877.png">

pls, some help.

Thanks!

**Comment by samus-aran** [comment]
Hi @junior-qdls  👋!

Once you become a verified student you have access to [Global Campus](https://education.github.com/globalcampus/student). 

GitHub Global Campus is a portal from which you can access your GitHub Education benefits and resources, all in one place. On the GitHub Global Campus portal, students can:

- Connect with a local Campus Expert. For more information on campus experts, see "[About Campus Experts](https://education.github.com/students/experts)."
- **Explore and claim offers for free industry tools from the [Student Developer Pack](https://education.github.com/pack).**
- See upcoming in-person and virtual events for students, curated by GitHub Education and student leaders.
- View assignments from [GitHub Classroom](https://classroom.github.com/) with upcoming due dates.
- Stay in the know with [Campus TV](https://www.twitch.tv/githubeducation) episodes. Campus TV is created by GitHub and student community leaders and can be watched live or on demand.
- Discover and contribute to student-created repositories from GitHub Community Exchange. For more information, see "[About GitHub Community Exchange](https://docs.github.com/en/education/explore-the-benefits-of-teaching-and-learning-with-github-education/github-global-campus-for-students/about-github-community-exchange)."

Let us know if the above links are working for you as a verified student ☝️ ⭐

**Comment by IBRAHIMtaiyeroheemat** [comment]
How did you verify yours? Because I've not been able to verify mine
